src/calendars/vietnam.ts

```diff
@@ -7,20 +7,74 @@ import { RomcalConfig, RomcalConfigInCalendarDef } from '@romcal/models/config/c
 import { Ranks } from '@romcal/constants/ranks/ranks.enum';
 import { Titles } from '@romcal/constants/titles/titles.enum';
 
-const defaultConfig: RomcalConfigInCalendarDef | undefined = undefined;
+const defaultConfig: RomcalConfigInCalendarDef = {
+  ascensionOnSunday: true,
+  corpusChristiOnSunday: true,
+  epiphanyOnSunday: true,
+};
 
 const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> => {
   const year = config.year;
   const _dates: Array<LiturgicalDayInput> = [
     {
       key: 'andrew_dung_lac_priest_and_companions_martyrs',
-      rank: Ranks.FEAST,
-      date: dayjs.utc(`${year}-11-13`),
+      rank: Ranks.SOLEMNITY,
+      date: dayjs.utc(`${year}-11-24`),
       liturgicalColors: LiturgicalColors.RED,
       metadata: {
         titles: [Titles.MARTYR],
       },
     },
+    {
+      key: 'joseph_spouse_of_mary',
+      rank: Ranks.SOLEMNITY,
+      date: dayjs.utc(`${year}-03-19`),
+      liturgicalColors: LiturgicalColors.WHITE,
+      metadata: {},
+    },
+    {
+      key: 'blessed_andrew_of_phu_yen',
```

Comment:
You did not define `blessed_andrew_of_phu_yen` in `en.ts`, therefore it would not be localised at all.

Also consider creating Vietnamese localisation.